### ISSUE TITLE
After upgrading from 0.3.1 to 0.3.2, all directories show as modified

### ISSUE THREAD

**Issue by cdown** [odeke-em/drive#471]
I just upgraded from 0.3.1 to 0.3.2, and on `drive pull` now all directories show as modified, when they in fact were not. A sample of my output:

```
M /Images
M /Images/2015-08
M /Images/2015-08/Peakop
M /Images/2015-07
M /Images/2015-07/MQ
```

The end then says:

```
Modification count 255 src: 816.11KB dest: 838.16KB
Addition count 2 src: 20.70MB
Proceed with the changes? [Y/n]:
```

The addition count appears to be correct, but no modifications to those directories took place, as far as I know. For now I've not updated since I'm worried about what might happen.

Any ideas?

```
% drive version
drive version: 0.3.2
Commit Hash: '848497f4ba68050820f32e1e54266e1eba2d5189'
Go Version: go1.5.1
OS: linux/amd64
BuildTime: 2015-11-09 11:10:14.928305808 +0000 GMT
```

**Comment by odeke-em** [comment]
Hello again @cdown,
Yes that's right there was a bug previously that prevented the match of ModTimes with remote that I noticed after `drive touch folders && drive pull folders` in which the pull was always ignored. I noticed that in issue https://github.com/odeke-em/drive/issues/463. This meant that you did a `drive stat` as well as `ls -l` on the same folders, the modTimes differed before last Friday's commit.

For now that's gonna be a seemingly spurious change but better to get all the times matching now than later. It is necessary later on to help with conflict detection.
